### PR TITLE
#1oyl5x1g Readme

### PR DESCRIPTION
**Pull request by Ally4**
#### What does this PR do?

- This PR is to update the readme

#### Description of Task to be completed?

- Creation of .coveralls.yml

- Update of coveralls script

#### How should this be manually tested?

- N/A

#### Any background context you want to provide?

- N/A

#### What are the relevant pivotal tracker stories?

- https://trello.com/c/1oYL5X1g/18-setup-continuous-integration-quality-assurance-pipeline

#### Screenshots (if appropriate)

- N/A

#### Questions:

- N/A

### CODE REVIEW

**Review by Mugema123 — APPROVED** [review]

**Review by TuyizeeAnastase — APPROVED** [review]

**Review by Ntwali-Josue — APPROVED** [review]

File: src/test/registration.test.js [hunk 1]
```diff
@@ -38,61 +38,4 @@ describe('Tests for the registration of the users', () => {
         done(error);
       });
   });
-  it('Should not be possible to do any task if not an admin', (done) => {
```
Comment (blaise82) [inline src/test/registration.test.js]: why did you delete these tests?

Comment (Ally4) [reply src/test/registration.test.js]: Because we don't really need tests on readme and they were taking very long while this PR has to be the first to be merged for the others to be merged too.

Comment (blaise82) [reply src/test/registration.test.js]: they are registration tests not about read me.
and they were there before
